go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/vmware/go-vcloud-director/v2
 
 require (
 	github.com/araddon/dateparse v0.0.0-20190622164848-0fb0a474d195
+	github.com/davecgh/go-spew v1.1.0
```

Comment:
Why do we need, which function is using it?

Reply:
thank you. removed - was used for testing